src/stories/avatar/index.tsx

```diff
@@ -2,13 +2,12 @@ import { Avatar as ZendeskAvatar } from "@zendeskgarden/react-avatars";
 import { theme } from "../theme";
 import styled from "styled-components";
 import { AvatarArgs } from "./_types";
-// import useWindowSize from "../../hooks/useWindowSize";
 
 const UgAvatar = styled(ZendeskAvatar)<AvatarArgs>`
   ${(props) =>
-    props.avatarType &&
-    props.avatarType !== "image" &&
-    `background: ${props.backgroundColor || theme.gradients.dark};`}
+    `background: ${
+      props.backgroundColor || theme.gradients.dark
+    }; text-transform: uppercase;`}
 `;
```

Comment:
Nel caso di Avatar di tipo immagine, non ci deve essere background, in quanto genera uno sgradevole bordino.